scriptworker/log.py

```diff
@@ -51,13 +51,6 @@ def update_logging_config(context, log_name=None, file_name='worker.log'):
 
     # Rotating log file
     makedirs(context.config['log_dir'])
-    path = os.path.join(context.config['log_dir'], file_name)
-    handler = logging.handlers.RotatingFileHandler(
-        path, maxBytes=context.config['log_max_bytes'],
-        backupCount=context.config['log_num_backups'],
-    )
-    handler.setFormatter(formatter)
-    top_level_logger.addHandler(handler)
```

Comment:
This change will stop all scriptworker logging to logfiles; we'll only get log output to the console.
I'm pretty sure that's not intended :)

We would want to make sure we had logrotate.d rotation with all scriptworker instances first. We're missing signing, pushapk, and transparency atm.

We might want to switch to [WatchedFileHandler](https://docs.python.org/2/library/logging.handlers.html#watchedfilehandler) since it references `logrotate`.

With that change, we probably want to remove [`log_max_bites` and `log_num_backups`](https://github.com/mozilla-releng/scriptworker/blob/master/scriptworker/constants.py#L39-L40) from `scriptworker/constants.py`. If those were specified in puppet, we'd need to remove them [here](https://hg.mozilla.org/build/puppet/file/tip/modules/scriptworker/templates/scriptworker.yaml.erb) (but they're not, so we're good there).